src/arrayMethodJoin.js

```diff
@@ -4,9 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let str = '';
```

Comment:
Try to give more clearly name for your variables
```suggestion
    let newString = '';
```